Okta.AspNet.Abstractions/StrictSecurityTokenValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿// <copyright file="StrictSecurityTokenValidator.cs" company="Okta, Inc">
+// Copyright (c) 2018-present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Okta.AspNet.Abstractions
+{
+    public sealed class StrictSecurityTokenValidator : ISecurityTokenValidator
+    {
+        private readonly JwtSecurityTokenHandler _handler;
+
+        public StrictSecurityTokenValidator(OktaWebOptions options)
```

Comment:
nit: Do we need the `options` constructor parameter? Seems like it isn't used.

Reply:
good catch, this is not needed.